hazelcast/src/test/java/com/hazelcast/config/InvalidConfigurationTest.java

```diff
@@ -109,13 +123,15 @@ public void testWhenValid_MapEvictionPolicy() {
         buildConfig("map-eviction-policy", "NONE");
     }
 
-    @Test(expected = InvalidConfigurationException.class)
+    @Test
     public void testWhenInvalid_MapEvictionPercentage() {
+        expectInvalid(" Value '101' is not facet-valid with respect to maxInclusive '100'");
```

Comment:
Personally i'm never crazy about checking exception messages. But in this case it is probably the best thing to do.